deepchem/feat/__init__.py

```diff
@@ -34,7 +34,8 @@
 from deepchem.feat.molecule_featurizers import RDKitDescriptors
 from deepchem.feat.molecule_featurizers import SmilesToImage
 from deepchem.feat.molecule_featurizers import SmilesToSeq, create_char_to_idx
-from deepchem.feat.molecule_featurizers import MATFeaturizer
+from deepchem.feat.molecule_featurizers import RobertaFeaturizer
```

Comment:
Would it be prudent to actually take this out of molecule_featurizers and make RobertaFeaturizer a general featurizer in feat?

Reply:
It would be quite trivial to subclass this class in molecule_featurizers to make it handle molecules, and in my mind that would make for a cleaner inheritance model

Reply:
hmm - fair in the case that this is applicable to any vocab beyond molecules - @rbharath thoughts on this?

Reply:
+1 for making this a more general featurizer since we can use it for other non-small-molecule applications